pkg/migration/validate_resource.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright © 2021 The Persistent-Volume-Migrator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package migration
+
+import (
+	"context"
+	"fmt"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8s "k8s.io/client-go/kubernetes"
+)
+
+// validateResources checks if required areguments exists
+func validateResources(client *k8s.Clientset, sourceSC, destinationSC, rookNS, cephClusterNS string) error {
+	getOpt := v1.GetOptions{}
+	ctx := context.TODO()
+
+	_, err := client.StorageV1().StorageClasses().Get(ctx, destinationSC, getOpt)
+	if err != nil {
+		return fmt.Errorf("Destination StorageClass name %s doesn't exists. %v ", destinationSC, err)
+	}
```

Comment:
```suggestion
_, err := client.StorageV1().StorageClasses().Get(ctx, sourceSC, getOpt)
if err != nil {
if error is not found{
			return fmt.Errorf("soure StorageClass %v doesn't exists. %v ", sourceSC, err)
		}
// return failed to get storageclass
}
```

Reply:
implement same check for other resources also

Reply:
what other resources? It is checking if all the CLI arguments that are being passed exist.

Reply:
i meant to do same error not found check for sourceSC,rookNS and cephclusterNS

Reply:
maybe I'm missing something but check for `sourceSC,rookNS and cephclusterNS` is already there. They are just inside if just to check if they are not empty.

Reply:
what i mean to say is add not found check to those. currently you are blindly returning not found error in case of any error

Reply:
ahh got it. Added. Thanks